arrow-libs/core/arrow-core-data/src/main/kotlin/arrow/core/computations/either.kt

```diff
@@ -11,14 +11,14 @@ fun interface EitherEffect<E, A> : Effect<Either<E, A>> {
 
   suspend fun <B> Either<E, B>.bind(): B =
     when (this) {
-      is Either.Right -> b
-      is Either.Left -> control().shift(this@bind)
+      is Either.Right -> value
+      is Left -> control().shift(this@bind)
```

Comment:
```suggestion
      is Either.Left -> control().shift(this@bind)
```
**Nit:** Just to be consistent with the other branch in the pattern matching